ruyi/cli/cmd.py

```diff
@@ -1,7 +1,9 @@
 import argparse
 from typing import Callable, IO, Protocol, TYPE_CHECKING
+import argcomplete
```

Comment:
External non-stdlib imports should get separated from the stdlib imports by a newline.

```suggestion

import argcomplete
```